Cargo.toml

```diff
@@ -1,6 +1,9 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 [workspace]
+# This is needed to avoid pulling in tokio features in wasm targets, due to new features in the `worker` library
```

Comment:
```suggestion
# This is needed to avoid pulling in tokio features in wasm targets, due to new features in version 0.0.18 of the `worker` crate
```